src/admin.controller.ts

```diff
@@ -111,6 +116,7 @@ export class DefaultAdminController {
       metadata,
       page,
       resultsPerPage,
+      listDisplay,
```

Comment:
pass the entire adminEntity, the templates are responsible to use the few objects they have available (adminSite, adminEntity...)